shared-libs/cht-datasource/test/local/libs/lineage.spec.ts

```diff
@@ -16,12 +16,12 @@ describe('local lineage lib', () => {
   beforeEach(() => {
     debug = sinon.stub(logger, 'debug');
   });
-
+  //
   afterEach(() => sinon.restore());
 
   it('getLineageDocsById', async () => {
     const uuid = '123';
-    const queryFn = sinon.stub().returns(Promise.resolve([]));
+    const queryFn = sinon.stub().returns([]);
```

Comment:
```suggestion
    const queryFn = sinon.stub().resolves([]);
```

Minor, but this is technically more correct since the real version of this function will return a promise.  I guess the test will pass either way since the `await` on line 31 does not really care if you give it a function or not.